src/main/java/chylex/customwindowtitle/TitleConfig.java

```diff
@@ -1,86 +1,115 @@
 package chylex.customwindowtitle;
 
-import com.google.common.collect.ImmutableMap;
-import com.google.common.collect.ImmutableSet;
 import java.io.IOException;
-import java.nio.charset.StandardCharsets;
+import java.io.InputStream;
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.Paths;
+import java.nio.charset.StandardCharsets;
+import java.util.Collections;
 import java.util.LinkedHashMap;
 import java.util.Map;
 import java.util.stream.Collectors;
 
 public final class TitleConfig {
-	private static final ImmutableMap<String, String> DEFAULTS = ImmutableMap.<String, String>builder()
-		.put("title", "Minecraft {mcversion}")
-		.build();
-	
-	private static final ImmutableSet<String> IGNORED_KEYS = ImmutableSet.of(
-		"icon16",
-		"icon32"
-	);
-	
+	private static final Map<String, String> DEFAULTS;
+	private static TitleConfig instance;
+
+	static {
+		final Map<String, String> defaults = new LinkedHashMap<>();
```

Comment:
Reason for this change?

Reply:
You can argue about this change, I thought it was better to not use guava here, because it's not necessary in this situation. The partial immutability is enough in this case. In my opinion using a builder here is quite overkill.